tests/cli/build/plugin_tests.py

```diff
@@ -0,0 +1,259 @@
+import os
+import unittest
+
+
+from core.base_test.tns_test import TnsTest
+from core.enums.os_type import OSType
+from core.enums.platform_type import Platform
+from core.settings import Settings
+from core.utils.device.adb import Adb
+from core.utils.file_utils import File, Folder
+from core.utils.npm import Npm
+from data.apps import Apps
+from products.nativescript.tns import Tns
+
+
+class CreateTests(TnsTest):
+    app_name = Settings.AppName.DEFAULT
+    app_path = os.path.join(Settings.TEST_RUN_HOME, 'data', 'temp', 'TestApp')
+
+    @classmethod
+    def setUpClass(cls):
+        TnsTest.setUpClass()
+        Tns.create(cls.app_name, app_data=Apps.HELLO_WORLD_JS, update=False)
```

Comment:
If we don't need something webpack or other specific template things we can use `Apps.MIN_JS`.